python.d/dnsdist.chart.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+from json import loads
+from bases.FrameworkServices.UrlService import UrlService
+
+ORDER = ['latency_rel', 'latency_abs', 'cache_rel', 'cache_abs', 'acl_abs', 'invalid_abs', 'queries_rel', 'queries_abs', 'health_abs']
+CHARTS = {
+	'latency_rel': {
+		'options': [None, 'Average latency', 'value', 'Latency', 'dnsdist.latency_rel', 'area'],
```

Comment:
family should be in lowercase (`Latency`). chart type should be `stacked` imo.

Reply:
Area looked more pretty to me than stacked but this is only my personal pref